### ISSUE TITLE
Add the Batch Permission Change prompt to Actor & Scene Folders

### ISSUE THREAD

**Issue by JayBWalker** [wsaunders1014/BatchPermissionsByFolder#2]
As titled. It would be fantastic if this module could be expanded to perform the same for all folder types. Cheers.

**Comment by ColombianJulio** [comment]
I agree. Pokemon 5e is very popular and the idea to have them use more than one character sheet, this would be helpful.
